Mimir/src/models/PlayerStat.php

```diff
@@ -200,6 +200,92 @@ protected function _getPlacesSummary(int $playerId, array $games)
+     * @param int $han
+     * @param int|null $fu
+     * @param int $winnerId
+     * @param int|null $loserId
+     * @param int[] $riichiIds
+     * @param int $playerId
+     * @param int $dealerId
+     * @param int $honba
+     * @param int $riichiBetsCount
+     * @param bool $isTsumo
+     * @param int|null $paoPlayerId
+     * @param int|null $closestWinnerForMultiron
+     * @param int $totalRiichiInRoundForMultiron
+     * @return int
+     * @throws InvalidParametersException
+     */
+    protected function _getPointsDelta(\Common\Ruleset $rules, array $currentScores, int $han, ?int $fu, int $winnerId, ?int $loserId, array $riichiIds, int $playerId, int $dealerId, int $honba, int $riichiBetsCount, bool $isTsumo, ?int $paoPlayerId, ?int $closestWinnerForMultiron = null, int $totalRiichiInRoundForMultiron = 0)
+    {
+        if ($isTsumo) {
+            $newScores = PointsCalc::tsumo($rules, $dealerId, $currentScores, $winnerId, $han, $fu, $riichiIds, $honba, $riichiBetsCount, $paoPlayerId);
+        } else {
+            /** @var int $nonnullLoserId */
+            $nonnullLoserId = $loserId;
+            $isDealer = $dealerId == $winnerId;
+            $newScores = PointsCalc::ron($rules, $isDealer, $currentScores, $winnerId, $nonnullLoserId, $han, $fu, $riichiIds, $honba, $riichiBetsCount, $paoPlayerId, $closestWinnerForMultiron, $totalRiichiInRoundForMultiron);
+        }
+        return $newScores[$playerId] - $currentScores[$playerId];
+    }
+
+    /**
+     * @param RoundPrimitive $r
+     * @param int $playerId
+     * @param bool $isTsumo
+     * @param array|null $multironRiichiWinners
+     * @return int
+     * @throws EntityNotFoundException
+     * @throws InvalidParametersException
+     */
+    protected function _getPointsDeltaForRound(RoundPrimitive $r, int $playerId, bool $isTsumo, ?array $multironRiichiWinners = null)
+    {
+        $sessionState = $r->getLastSessionState();
+        $winnerId = $r->getWinnerId();
+        $riichiIds = $r->getRiichiIds();
+
+        if ($multironRiichiWinners !== null) {
+            $closestWinnerForMultiron = $multironRiichiWinners[$winnerId]['closest_winner'];
+            $totalRiichiInRoundForMultiron = count($riichiIds);
+            $riichiIds = $multironRiichiWinners[$winnerId]['from_players']; // overwrite for one ron instance of multiron
+            $honba = $multironRiichiWinners[$winnerId]['honba'];
+            $riichiBetsCount = $multironRiichiWinners[$winnerId]['from_table'];
+        } else {
+            $closestWinnerForMultiron = null;
+            $totalRiichiInRoundForMultiron = 0;
+            $honba = $sessionState->getHonba();
+            $riichiBetsCount = $sessionState->getRiichiBets();
+        }
+
+        if ($winnerId != $playerId) {
+            // if we deal-in or someone else takes tsumo, don't consider any riichi sticks in the current round
+            // since we are interested only in deal-in cost
+            $riichiIds = array();
```

Comment:
There's shortcut that just works: `$riichiIds = []`

Reply:
Done